app/lib/proxy_api/pulp.rb

```diff
@@ -0,0 +1,19 @@
+module ProxyAPI
```

Comment:
In the Foreman code base, these lives under 'lib/proxy_api' -- could we move it to there in this code base for consistency?